src/iolStateMachineHandler/src/module.cpp

```diff
@@ -480,20 +480,20 @@ void Manager::drawScoresHistogram(const Bottle &blobs,
             {
                 if (imgTmp2_width>imgConf.width()/4)
                 {
-                    double r=imgTmp2_height/imgTmp2_width;
+                    double r=(double)imgTmp2_height/(double)imgTmp2_width;
                     imgTmp2_width=imgConf.width()/4;
                     imgTmp2_height=(size_t)(r*imgTmp2_width);
                 }
             }
             else if (imgTmp2_height>imgConf.height()/4)
             {
-                double r=imgTmp2_width/imgTmp2_height;
+                double r=(double)imgTmp2_width/(double)imgTmp2_height;
                 imgTmp2_height=imgConf.height()/4;
                 imgTmp2_width=(size_t)(r*imgTmp2_height);
             }
 
             ImageOf<PixelBgr> imgTmp2;
-            imgTmp2.resize(imgTmp2_width,imgTmp2_height);
+            imgTmp2.resize(max((size_t)1,imgTmp2_width),max((size_t)1,imgTmp2_height));
```

Comment:
I would make explicit use of `std` scope for `max` function, as done elsewhere I think.